buildenv/jenkins/openjdk_tests

```diff
@@ -144,17 +151,18 @@ def PLATFORM_MAP = [
     'x86-64_windows_mixed' : [
         'SPEC' : 'win_x86-64_mxdptrs',
         'LABEL' : 'ci.role.test&&hw.arch.x86&&sw.os.windows',
+        'DynamicAgents' : ['fyre']
     ],
     'x86-64_windows_xl' : [
         'SPEC' : 'win_x86-64',
         'LABEL' : 'ci.role.test&&hw.arch.x86&&sw.os.windows',
+        'DynamicAgents' : ['fyre']
```

Comment:
We cannot start Fyre vm agents on windows atm.

Reply:
Removed. Thanks.